src/featurizations/graphnodefeaturization.jl

```diff
@@ -112,10 +112,9 @@ function chunk_vec(vec::Vector{<:Real}, nbins::Vector{<:Integer})
     return chunks
 end
 
-function featurize!(ag::AtomGraph, fzn::GraphNodeFeaturization)
-    encoded = reduce(vcat, map((x) -> x(ag), fzn.features))
-    ag.encoded_features = encoded
-    ag.featurization = fzn
+function featurize(ag::AtomGraph, fzn::GraphNodeFeaturization)
```

Comment:
I think this should actually be an alias for the constructor of the new container object. Because otherwise it's not featurizing (i.e. attaching encoded features to a structure), it's just encoding...

Reply:
Isn't it doing that in the inner constructor of `SerializableEncodedFeature`, while the container object is being created?
Or am I misunderstanding your point here?

Reply:
Yes, this is purely a naming thing. If we want a function that does what this one does, I think it should be called something like `encode`, and a function called `featurize` should actually do the featurizing (i.e. attaching of encoded features to Atoms object), which would mean that it should essentially be an alias of the `FeaturizedAtoms` constructor. While redundant, I think it probably makes sense to have such a function because it's a pretty intuitive syntax that might be easier for a user to remember.

Reply:
Ah, got it. Slightly satisfying that we can now have both `decode` AND `encode` haha

Reply:
Resolved in 6a61bf5 and 676ee63.